src/docs/patterns/internationalization.mdx

```diff
@@ -13,42 +16,83 @@ import Image4 from '../../app/assets/design-patterns/internationalization/web-in
 
 Internationalization is the practice of adapting applications to various regions, languages, and cultures. You can accomplish this in your applications by considering the differences in language, scripts, and directionality.
 
-## Language Support
+<Divider />
 
-To localize your application content, you should create a string resource file or object that contains the translations for any display text used in your application. You will not want to have any text hard-coded in your application's UI. To switch your application to another language, you simply toggle the file/object used to populate the UI to the appropriate translation.
+# Steps for Converting from One Locale to Another Locale
 
-<MaterialDesignDescription
-    title={'Language Support'}
-    url={'https://material.io/design/typography/language-support.html#language-considerations'}
-/>
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Language Support'}
+        url={'https://material.io/design/typography/language-support.html#language-considerations'}
+    />
+    <MaterialDesignDescription
+        title={'Localization'}
+        url={'https://developer.apple.com/localization/'}
+        description={`Apple's suggestions on designing for an international market.`}
+        avatar={<Apple style={{ width: 48, height: 48 }} />}
+    />
+</React.Fragment>
 
-Different languages/writing systems may require different vertical and horizontal space. To ensure flexibility, avoid using fixed dimensions for necessary UI components such as buttons, labels, and badges. Also, consider setting different default display font sizes for languages that may have smaller glyphs.
+## Showing Language Options
 
-> **NOTE:** PX Blue recommends using [Noto Sans](https://www.google.com/get/noto/#sans-lgc) for all non-western languages. You can read more on our [Typography](/style/typography) page.
+When showing a list of languages, you can decide whether you want to translate the language names or not (e.g., "English" vs. "Inglés"). There are pros and cons to each approach, so consider what makes the most sense for your application. Providing other cues (such as an icon) to help users find the language selection can help.
 
-### Showing Language Options
+## Translating the Text
 
-When showing a list of languages, you can decide whether you want to translate the language names or not (e.g., "English" vs. "Inglés"). There are pros and cons to each approach, so consider what makes the most sense for your application. Providing other cues (such as an icon) to help users find the language selection can help.
+To localize your application content, you should create a string resource file or object that contains the translations for any display text used in your application. You will not want to have any text hard-coded in your application's UI. To switch your application to another language, you simply toggle the file/object used to populate the UI to the appropriate translation.
```

Comment:
localization = address the **_cultural_** difference, such as the tone, the images, etc, more about marketing (so usually applicable to physical product design and web design, but less about application design)

internationalization = everything else, so that you can render different languages, units, currencies... without breaking your UI